compat.h

```diff
@@ -0,0 +1,37 @@
+#ifndef UTCP_COMPAT_H
+#define UTCP_COMPAT_H
+
+#ifndef _MSC_VER
+  #include <unistd.h>
+#endif
+
+#ifdef _WIN32
+  #include <winsock2.h>
+#else
+  #include <sys/time.h>
+#endif
+
+#if !defined(PRINT_SIZE_T)
```

Comment:
ich änder das in #ifndef, !defined nutze ich eigentlich nur bei mehreren prüfungen
#ifndef wird an zuvielen stellen im code genutzt als dass ich das jetzt alles in !defined ändern wollte :)